tests/live/test_workflow.py

```diff
@@ -19,7 +20,14 @@ def warn(msg):
     LOGGER.warning(msg)
 
 
-def run(cmd, log_cmd=False) -> dict:
+def error(msg):
+    LOGGER.error(msg)
+
+
+def run(
+    cmd,
+    log_cmd=False,
+) -> CompletedProcess[Any] | CompletedProcess[bytes] | CompletedProcess[str]:
```

Comment:
I got a complaint from the IDE that the return error was wrong, so I let it fix it.